docs/command-reference/sorted-sets/zscan.md

```diff
@@ -6,7 +6,7 @@ description: Incrementally iterate sorted sets elements and associated scores
 
 ## Syntax
 
-    ZSCAN key cursor
+    ZSCAN key cursor [MATCH pattern] [COUNT count]
```

Comment:
Consider adding something on those options (I know redis do not have them)

Reply:
there is a long description on them in th scan documentation
In the bottom it says see SCAN doc